examples/multi_agent/MANet.py

```diff
@@ -0,0 +1,29 @@
+import torch
+import numpy as np
+from torch import nn
+
+from tianshou.data import to_torch
+
+
+class Net(nn.Module):
+    def __init__(self, layer_num, state_shape, action_shape=0, device='cpu',
+                 softmax=False):
+        super().__init__()
+        self.device = device
+        self.model = [
+            nn.Linear(np.prod(state_shape), 128),
+            nn.ReLU(inplace=True)]
+        for i in range(layer_num):
+            self.model += [nn.Linear(128, 128), nn.ReLU(inplace=True)]
+        if action_shape:
+            self.model += [nn.Linear(128, np.prod(action_shape))]
+        if softmax:
+            self.model += [nn.Softmax(dim=-1)]
+        self.model = nn.Sequential(*self.model)
+
+    def forward(self, s, state=None, info={}):
+        s = to_torch(s, device=self.device, dtype=torch.float)
```

Comment:
I would rather explicit `torch.float32` or `torch.float64` since it is a convention specific to each library.

Reply:
The code is copied from other examples. Maybe this can be fixed by another PR.

Reply:
I don't get why to be in a hurry. Why is big deal to do it now ? Since it is not even a required feature on `master` branch.

Reply:
I'm working on this by another pr.

Reply:
Do as you wish, but keep in mind you don't have my approval.

Reply:
> I don't get why to be in a hurry. Why is big deal to do it now ? Since it is not even a required feature on `master` branch.

I want to keep each PR's integrity by focusing on one theme :)

Reply:
Solved in 185f5dc07bf874d6be63307b6f010f9b8e2d7ad4 . This file will be removed.

Reply:
> I want to keep each PR's integrity by focusing on one theme :)

In my opinion, one theme is not: release something that has flaws easy to fix

But it has already been discussed anyway.